src/xava.c

```diff
@@ -224,6 +224,22 @@ as of 0.4.0 all options are specified in config file, see in '/home/username/.co
         thing.xava = &xava;
         xavaIONotifyAddWatch(&thing);
 
+        // attach pywal colors file to the IONotify thing
+        static const char b[] = "/.cache/wal/colors";
+        int lena = strlen(getenv("HOME"));
+        int lenb = strlen(b);
+        char *filename = malloc(lena+lenb+1);
+        // copy & concat (including string termination)
+        memcpy(filename,getenv("HOME"),lena);
```

Comment:
I know I may come out as not-C-like but I'd like to avoid doing pointer math as much as possible.